### DISCUSSION TITLE
有没有办法爬取博主显示设置之前（近半年可见等）的微博？ 观察到博主转发时间限制之前的微博可以看到里面的全部内容。

### DISCUSSION THREAD

**Discussion by douhenhuidyf** [Chilfish/Weibo-archiver#59] [功能请求]
有没有办法爬取博主显示设置之前（近半年可见等）的微博？ 观察到博主转发时间限制之前的微博可以看到里面的全部内容，有没有办法模拟从而爬下来？

**Comment by Chilfish** [comment]
有无例子？毕竟这个半年可见的时间段应该是根据微博服务器时间来判断的，我们用户端并不能控制。

插件也只能获取到用户端在网页能看到的样子，如果获取时已经看不到的微博就获取不了了：

![image](https://github.com/user-attachments/assets/c9a87711-6514-4038-bcb6-85e13a4d2554)
